src/Avalonia.OpenGL/AngleOptions.cs

```diff
@@ -12,7 +12,7 @@ public enum PlatformApi
 
         public List<PlatformApi> AllowedPlatformApis = new List<PlatformApi>
         {
-            PlatformApi.DirectX9
+            PlatformApi.DirectX11
```

Comment:
I think AllowedPlatformApis should be null here

Reply:
The default is here: https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.OpenGL/EglDisplay.cs#L38 and should then be changed to DX11

Reply:
ok changed.

Reply:
Also, and this is kinda off-topic, but why is it a `List<T>`? A `ReadOnlyList<T>` or even a `List<T>` would allow you to use an array instead of allocating a list.

Reply:
made it an array and used IList<T>